### ISSUE TITLE
tilelive.auto doesn't work with scoped packages

### ISSUE THREAD

**Issue by mapsam** [mapbox/tilelive#187]
the `tilelive-copy` command uses `tilelive.auto`, which tries to load the plugin module automatically by concatenating strings from the protocol. Loading a scoped npm module, such as `@mapbox/tilelive-omnivore` doesn't work, since the attempted require looks like `require('tilelive-omnivore')` and this [fails silently](https://github.com/mapbox/tilelive/blob/a66c37db776cd01d6241c1d25da0cf7c7846fa9a/lib/tilelive.js#L121-L122), leading to an error in the actual copy operation.

Unfortunately we can't update `tilelive.auto` to work with scoped packages, since we aren't sure what the name of the scope package is (i.e. can't assume it will always be `@mapbox`).

A solution would be to provide an added flag to the CLI called `--protocol` (name TBD) that allows you to pass in a string of whatever protocol you want to register - this should match the name of the module in the package.json. Example:

```
tilelive-copy omnivore://path/file.geojson --protocol @mapbox/tilelive-omnivore
```

cc @rclark @perrygeo @GretaCB

**Comment by rclark** [comment]
:+1: Probably it should've always been this way and `.auto` shouldn't have existed. 

I would call it either `--register` or `--register-module`.

**Comment by jingsam** [comment]
Fixed in #191

**Comment by mapsam** [comment]
Thanks @jingsam!
